statsmodels/base/_penalties.py

```diff
@@ -110,13 +110,190 @@ def func(self, params):
         v = np.sqrt(1 + (params / self.dlt)**2)
         v -= 1
         v *= self.dlt**2
-        return np.sum(self.wts * self.alpha * v)
+        return np.sum(self.wts * self.alpha * v, 0)
 
     def grad(self, params):
         v = np.sqrt(1 + (params / self.dlt)**2)
         return params * self.wts * self.alpha / v
 
 
+class SCAD(Penalty):
+    """
+    The SCAD penalty of Fan and Li
```

Comment:
document tau and c
i.e, add formula for SCAD
Note, I'm not using `alpha` as in super class